src/lib/components/table/table-settings/dialog/table-settings-dialog.glob.scss

```diff
@@ -0,0 +1,61 @@
+mat-dialog-content
```

Comment:
Diese Styles sollten in irgendeiner Form gescoped werden.
Da diese Datei in unserer `styles.scss` importiert wird, sind es globale styles, die sich so auf jeden MatDialog und jede MatList im Dialog auswirken würden.

Reply:
Eine spezielle Klasse auf der Liste oder dem DialogContent würde z.B. schon helfen.